frontend/src/apiMocks/mockData.ts

```diff
@@ -329,5 +329,31 @@ export const mockedInboundContacts: Record<
       num_texts_blocked: 18,
       blocked: false,
     },
+    {
```

Comment:
I added these to show the "Yesterday at 13:37" lines. (Which are no longer shown as such after this PR.)